app/assets/javascripts/directives/views/editorMenu.ts

```diff
@@ -43,14 +39,15 @@ class EditorMenuCtrl extends PureViewCtrl implements EditorMenuScope {
 
   $onInit() {
     super.$onInit();
-    const editors = this.application.componentManager!.componentsForArea(ComponentArea.Editor)
+    const editors = this.application
+      .componentManager!.componentsForArea(ComponentArea.Editor)
       .sort((a, b) => {
         return a.name.toLowerCase() < b.name.toLowerCase() ? -1 : 1;
       });
     const defaultEditor = editors.filter((e) => e.isDefaultEditor())[0];
     this.setState({
       editors: editors,
-      defaultEditor: defaultEditor
+      defaultEditor: defaultEditor,
```

Comment:
This line can be deleted it seems.

Reply:
The `isEditorDefault()` function can be removed then as well?

Reply:
Yes doesn't look like this menu needs anything to do with defaults anymore.